data/wanderer/wanderers middle.txt

```diff
@@ -3302,13 +3321,16 @@ event "wanderers: sabira eseskrai colony"
 		description `	Due to a recent shift in climate the deserts have begun shrinking once more, with local vegetation growing to cover more and more land each year. The Wanderers, working together with the Kor Efreti, have founded a settlement here and begun working to accelerate the process of rejuvenating the wastelands.`
 		spaceport `This small settlement is populated by a roughly equal number of Korath and Wanderer settlers. Since most of them do not speak each other's language, the two groups do not mingle much, and communicate mostly via gestures and drawings. It's not much, but it's at least a start for the two species to begin learning from each other.`
 
-mission "Wanderers: Sestor: Final"
+mission "Wanderers: Sestor: Final: Patched"
```

Comment:
We have like four patch missions down here now. Can we not clean some of these up?